cmd/export/cluster.go

```diff
@@ -0,0 +1,244 @@
+package export
+
+import (
+	"fmt"
+	"strings"
+
+	authv1 "github.com/openshift/api/authorization/v1"
+	securityv1 "github.com/openshift/api/security/v1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type ClusterScopeHandler struct {
+}
+
+type admittedResource struct {
+	APIgroup string
+	Kind     string
+}
+
+var admittedClusterScopeResources = []admittedResource{
+	{Kind: "ClusterRoleBinding", APIgroup: "rbac.authorization.k8s.io"},
+	{Kind: "ClusterRole", APIgroup: "rbac.authorization.k8s.io"},
+	{Kind: "SecurityContextConstraints", APIgroup: "security.openshift.io"},
+}
+
+func NewClusterScopeHandler() *ClusterScopeHandler {
+	clusterScopeHandler := &ClusterScopeHandler{}
+
+	return clusterScopeHandler
+}
+
+func isClusterScopedResource(apiGroup string, kind string) bool {
```

Comment:
```suggestion
func isAdmittedClusterScopedResource(apiGroup string, kind string) bool {
```

Reply:
Maybe `isAdmitted` is even better, currently there is a mismatch between the name of the function and what it's doing.